### ISSUE TITLE
ReverseV2 is not supported

### ISSUE THREAD

**Issue by Zrufy** [onnx/tensorflow-onnx#905]
when i'm trying to convert my pb with opset = 9 i obtain this error

```
ERROR - Tensorflow op [bidirectional_1/ReverseV2: ReverseV2] is not supported
ERROR - Tensorflow op [bidirectional_1/ReverseV2_1: ReverseV2] is not supported
ERROR - Tensorflow op [bidirectional_2/ReverseV2: ReverseV2] is not supported
ERROR - Tensorflow op [bidirectional_2/ReverseV2_1: ReverseV2] is not supported
```

Layer gru caused this problem?

with opset 10 or 11 

```
ValueError: make_sure failure: shape of axis 1 is unknown

AttributeError: 'NoneType' object has no attribute 'get_node_by_output'
```

**Comment by guschmue** [comment]
what tf version was the model generated with ? I think with tf-1.15 we see this ReverseV2 being used so you'd need opset-10 or better.
Can you send us the stack trace for the failure on opset-10 ?

**Comment by Zrufy** [comment]
sorry for poor information.Tensorflow version is 1.14 the failure i obtain is 

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/ktf2onnx/tf2onnx/tfonnx.py", line 352, in tensorflow_onnx_mapping
    func(g, node, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/ktf2onnx/tf2onnx/onnx_opset/tensor.py", line 1396, in version_10
    , "ReverseSequence batch size for axis {} is unknown".format(axis))
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/ktf2onnx/tf2onnx/utils.py", line 290, in make_sure
    raise ValueError("make_sure failure: " + error_msg % args)
ValueError: make_sure failure: ReverseSequence batch size for axis 0 is unknown

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/stiv/lpr/convert_keras_to_uff.py", line 14, in <module>
    keras2onnx.convert_keras(model, model.name)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/main.py", line 102, in convert_keras
    parse_graph(topology, sess.graph, target_opset, output_names)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/parser.py", line 667, in parse_graph
    return _parse_graph_scope(graph, keras_layer_ts_map, topo, top_level, output_names)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/parser.py", line 626, in _parse_graph_scope
    _infer_graph_shape(topology, top_scope, varset)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/parser.py", line 389, in _infer_graph_shape
    _finalize_tf2onnx_op(topology, oop, varset)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/parser.py", line 359, in _finalize_tf2onnx_op
    g = tf2onnx_wrap(topo, subgraph, outputs, varset.target_opset)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/wrapper.py", line 32, in tf2onnx_wrap
    raise e
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/wrapper.py", line 26, in tf2onnx_wrap
    output_names=outputs)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/ktf2onnx/tf2onnx/tfonnx.py", line 571, in process_tf_graph
    mapped_op, unmapped_op, exceptions = tensorflow_onnx_mapping(g, ops_mapping)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/ktf2onnx/tf2onnx/tfonnx.py", line 355, in tensorflow_onnx_mapping
    logger.error("Failed to convert node %s\n%s", node.name, node.summary, exc_info=1)
  File "/usr/local/lib/python3.6/dist-packages/keras2onnx-1.5.2-py3.6.egg/keras2onnx/ktf2onnx/tf2onnx/graph.py", line 172, in summary
    node = g.get_node_by_output(name)
AttributeError: 'NoneType' object has no attribute 'get_node_by_output'
```

**Comment by jignparm** [comment]
It looks like the shape information for the node (at conversion time) is dynamic (i.e. -1).
https://github.com/onnx/tensorflow-onnx/blob/master/tf2onnx/onnx_opset/tensor.py#L1715

Will need to rewrite the algorithm/mapping to see if dynamic shape can be accepted, without making the operator too inefficient. Will update shortly on this.

**Comment by jignparm** [comment]
This should be fixed by #909.
